src/corg/development-tools-and-process/git-usage.md

```diff
@@ -819,9 +819,14 @@ curl -L https://github.com/nuxeo/nuxeo/pull/67.patch | git am
 #### As a Branch
 
 ```
-# Fetch is useless if you properly configured remote.origin.fetch at global or repository level
+# Fetch details are needed only if you didn't configure remote.origin.fetch at global or repository level
 git fetch origin +pull/67/head:pull/67
+
+# Pull requests can be checkouted like a standard branch
 git checkout pull/67
+
+# You can list the modified files and open them in your preferred IDE (here using Eclipse for a PR on master)
+git logone origin/master.. --name-only --pretty="format:" | xargs eclipse
```

Comment:
What about replacing `git logone` by `git log --oneline`?

Reply:
+1, please don't use too many custom git aliases in documentation.

Reply:
Done, it was actually not needed at all.

@mincong-h fyi, both commands are not equivalent, else I would have not created such a useless alias: `git logone` is `git log --pretty=format:'%C(auto)%m %h %Cgreen%ad %C(blue)%<(20)%aN %Creset%s %N %C(auto)%d' --left-right --cherry-pick --date=short`